homeassistant/components/vicare/sensor.py

```diff
@@ -270,6 +271,7 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         icon="mdi:percent",
         native_unit_of_measurement=PERCENTAGE,
         value_getter=lambda api: api.getModulation(),
+        device_class=SensorDeviceClass.POWER_FACTOR,
```

Comment:
The power factor device class represents this:
https://en.m.wikipedia.org/wiki/Power_factor

Is this the same measurement as the burner modulation?

Reply:
_Boiler modulation is the ability of a boiler to 'turn down' its output. That is to say if you have a 20kw boiler, but you only require 10kw of heat for the next hour, rather than outputting 20kw for 5 minutes then resting 5 minutes repeatedly, (how the older non-modulating boilers used to work) the boiler will simply turn down its flame by 50%._
https://www.heatgeek.com/what-is-boiler-modulation/

I think that fits the Wikipedia definition, right?

Reply:
I'm not an expert in this area but it doesn't sound to me as they represent the same thing. Here's another more hands on description of power factor:
https://www.fluke.com/en/learn/blog/power-quality/power-factor-formula

Reply:
Unfortunately I don't think I'll be able to find an answer to this. Since we have doubts let's remove this state class

Reply:
Opentherm modulation is just a percentage, not related to the electrical power factor at all.